.github/workflows/quest.yml

```diff
@@ -2,7 +2,7 @@ name: "quest import"
 on:
   issues:
     types:
-      [ labeled ]
+      [ labeled, closed, reopened, assigned, unassigned ]
```

Comment:
With these additional events, the logic currently on L18 needs to be updated.

Reply:
I'm not sure how, could you tell me what the rules are for that scenario? I thought the idea would be to only run when there is a trigger label present regardless.

Reply:
So, add the inquest label as part of the condition too - right?

Reply:
Updated